html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form id = 'coding-form'>
+     <div>Email:</div>
+     <label for="email">
+      <input type="email" id="email" name="user_email">
+     </label>
+
+     <div>Why is coding cool?</div>
+     <label for="reason">
+      <select id="reason">
+        <option value='1'>Select an option</option>
+        <option value='2'>Because it's interesting</option>
+        <option value='3'>Because it's fun</option>
+        <option value='4'>Because it's powerful</option>
+      </select>
+     </label>
+
+     <p> Favorite Language (choose one) </p>
+      <p>
+        <label for="HTML">
+          <input type="radio" id="HTML" name="language"> HTML
+        </label>
+        <label for="CSS">
+          <input type="radio" id="CSS" name="language"> CSS
+        </label>
+        <label for="JS">
+          <input type="radio" id="JS" name="language"> JavaScript
+        </label>
+      </p>
+
+      <p>Would you like to sign up for more information?</p>
+      <label for="signup">
+        <input type="checkbox" id="signup"> Sign me up!
+      </label>
+
+      <div>Comments</div>
+      <label for="comment">
+        <textarea cols="30" rows="8" id="comments" placeholder="Tell us your thoughts!"></textarea>
+      </label>
```

Comment:
`label` needs to have some text associated with it. Utilize your text content inside the `div` on line 43